client/rpc/status.go

```diff
@@ -61,7 +61,7 @@ func StatusCommand() *cobra.Command {
 				ValidatorInfo: validatorInfo{
 					Address:     status.ValidatorInfo.Address,
 					PubKey:      pk,
-					VotingPower: status.ValidatorInfo.VotingPower,
+					VotingPower: status.ValidatorInfo.StakingPower,
```

Comment:
ValidatorInfo.VotingPower is changed to ValidatorInfo.StakingPower by https://github.com/line/ostracon/commit/50b4e7834da84a1590847c9cc40c3aa2dbf97fb4#diff-18b278f1fcf823eee4511186b774e4d281c1274b2505685d59aa85f001c3e766